requirements-xpu.txt

```diff
@@ -3,10 +3,9 @@
 
 setuptools < 70.0.0 # IPEX's torch have some dependency. to be removed.
 
+ray >= 2.9
 torch == 2.3.1+cxx11.abi
 intel-extension-for-pytorch == 2.3.110+xpu
 oneccl_bind_pt == 2.3.100+xpu
 
 triton-xpu == 3.0.0b2
```

Comment:
shall we also leave one comment here to show the extra index url? 
--extra-index-url https://pytorch-extension.intel.com/release-whl/stable/xpu/us/